nom.go

```diff
@@ -181,7 +182,7 @@ func main() {
 			}
 		}
 
-		daysStarting31Dec1899, daysError := strconv.Atoi(record[fields["Order Create/Modification Date"]])
+		daysStarting31Dec1899, _ := strconv.Atoi(record[fields["Order Create/Modification Date"]])
```

Comment:
This changed because it's an ineffectual assignment, meaning that `daysError` isn't checked or used before it's reassigned later in the code.